### DISCUSSION TITLE
Dappwright init

### DISCUSSION THREAD

**Discussion by QAmetapro** [TenKeyLabs/dappwright#82] [General]
can anybody share the complete empty project using dappwright?
In the readme file there are missing imports, no idea what "base" is in "example.spec.ts" file - what we are going to extend
after adding imports i'm getting information like 
`'TS1192: Module '"/Users/.../node_modules/@tenkeylabs/dappwright/dist/index"' has no default export.'` 
and this is why does not see "dappwright" import
`import dappwright, {Dappwright, MetaMaskWallet} from "@tenkeylabs/dappwright";`

I'm new regarding js/ts so please forgive me :)

**Comment by osis** [comment]
Yes! This is something that has come up before. Traditional frameworks would have this enabled in their tsconfig https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports but newer frameworks are moving away from that pattern.

We'll make an issue to make sure this is tracked properly but for now, you can enable that option and it should be okay

**Reply by QAmetapro** [reply]
what about "base" ?

![image](https://user-images.githubusercontent.com/125270296/221239620-e3eb3a54-263e-40ad-a219-8c90f8e19820.png)

**Reply by osis** [reply]
`base` is an instance of `test`. That can be acquired via `import test as base from '@playwright/test';`
